cpp/collections_module/algorithm/collections.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <mgp.hpp>
+
+#include <string_view>
+
+namespace Collections {
+
+constexpr const std::string_view kProcedureSumLongs = "sumLongs";
```

Comment:
forgot about this, `sum_longs`

Reply:
Also, can you add comments so we can easily find which one is for what:

And group them
\* SumLongs constants *\
constexpr ....
....


\* Avg constants *\

constexpr....
...

void Avg ...
\\ rest of function declarations